examples/custom-mapping.php

```diff
@@ -10,7 +10,7 @@
 };
```

Comment:
Shouldn't this example make use of the `ExitAction` as callback instead of $menu->close() ?

Reply:
I don't think it matters, its just an example to show you can do whatever you want

Reply:
Yeah I just thought it would show that you can use those actions as part of a custom control too.
The only example given in the docs, they are used as addItem() callback.
But it isn't very important I agree, I was merely nitpicking.